kvmagent/kvmagent/test/shareblock_testsuite/test_shareblock_metadata.py

```
@@ -0,0 +1,1555 @@
+
+    # -- single entry point ---------------------------------------------------
+
+    @pytest_utils.ztest_decorater
+    def test_sblk_metadata_api(self):
+        iscsi_server = env.get_vm_metadata('self')
+        rsp = storage_device_utils.iscsi_login(
+            iscsi_server.ip, "3260"
+        )
+        self.assertEqual(rsp.success, True, "iscsiadm login failed")
+
+        r, o = bash.bash_ro("ls /dev/disk/by-id | grep scsi|awk -F '-' '{print $2}'")
```

Ruff (0.15.9)

[warning] Unpacked variable `r` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)